mappings/net/minecraft/SaveVersion.mapping

```diff
@@ -25,7 +25,8 @@ CLASS net/minecraft/class_6595 net/minecraft/SaveVersion
 		ARG 1 other
 	METHOD method_38494 getId ()I
 		COMMENT {@return the integer data version of this save version}
-	METHOD method_38495 hasOldWorldHeight ()Z
-		COMMENT This method always returns {@code false}, but its usage appears to
-		COMMENT indicate that it returns if this save version has the old 0 to 255 world
-		COMMENT height limit.
+	METHOD method_38495 hasNewWorldHeight ()Z
```

Comment:
Should be named `hasIncompatibleWorldHeight`. The translation key says "pre" "worldheight", but I would refer to the translation content in `en_us.json`, `Incompatible world height` for a somewhat better name.

Reply:
```suggestion
	METHOD method_38495 hasIncompatibleWorldHeight ()Z
```